src/pages/Map.js

```diff
@@ -32,45 +31,24 @@ function Map() {
 
   if (loadError) return 'Error loading maps';
   if (!isLoaded) return 'Loading maps';
```

Comment:
are there any other errors or cases we need to handle? I want to make sure we have all edge cases covered